app/src/main/java/net/skyscanner/backpack/demo/data/ExampleBpkCalendarController.kt

```diff
@@ -51,20 +49,17 @@ class ExampleBpkCalendarController(
 @VisibleForTesting
 internal fun multiColoredExampleCalendarColoring(
   colorOffset: Long,
-  startDate: CalendarDay,
-  endDate: CalendarDay,
+  startDate: LocalDate,
+  endDate: LocalDate,
   context: Context
 ): CalendarColoring {
-  val daysBetweenStartAndEnd = TimeUnit.DAYS.convert(endDate.date.time - startDate.date.time, TimeUnit.MILLISECONDS) + 1
-  val calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"))
-  calendar.set(Calendar.YEAR, startDate.year)
-  calendar.set(Calendar.MONTH, startDate.month)
-  calendar.set(Calendar.DAY_OF_MONTH, startDate.day)
-  val redSet = mutableSetOf<CalendarDay>()
-  val yellowSet = mutableSetOf<CalendarDay>()
-  val greenSet = mutableSetOf<CalendarDay>()
-  val greySet = mutableSetOf<CalendarDay>()
-  val emptySet = mutableSetOf<CalendarDay>()
+  val daysBetweenStartAndEnd = ChronoUnit.DAYS.between(startDate, endDate) + 1
+  val redSet = mutableSetOf<LocalDate>()
+  val yellowSet = mutableSetOf<LocalDate>()
+  val greenSet = mutableSetOf<LocalDate>()
+  val greySet = mutableSetOf<LocalDate>()
+  val emptySet = mutableSetOf<LocalDate>()
+  var dateIterator = LocalDate.now()
```

Comment:
This should be ` var dateIterator = LocalDate.of(startDate.year, startDate.month, startDate.dayOfMonth)
`

Reply:
Good catch. Fixed now